03-events-forms-refs-life-cycle/events/custom_link/src/App.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import './App.css';
+import CustomLink from './CustomLink'
+
+class App extends Component {
+   
+  constructor(props){
+    super(props);
+
+    this.state ={
+      url1: "https://www.pingmylinks.com/",
+      url2: "https://www.facebook.com/pingmylinks/",
+      url3: "https://www.excitesubmit.com/html/contact.html",
+      btn : "Disable Links"
+    }
+
+      this.handleClick = this.handleClick.bind(this);
+  }
+  
+
+    handleClick() {
+      if (this.state.url1 === "https://www.pingmylinks.com/"){
+         this.setState({
+             url1: "#",
+             url2: "#",
```

Comment:
I would also recommend to not replace the values of URLs, just handle this logic based on the value of `this.state.diabled`